### PR TITLE
Create README for paper

### PR DESCRIPTION
**Pull request by SabineHaas**
@MaGering would it be okay for you to add such a README to `with_battery/` , see changes?

**Changes proposed in this pull request**:
- Add README page for working paper
- minor corrections

The following steps were realized, as well (required):
- [ ] Update the CHANGELOG.md
- [ ] Check if full simulation tests pass locally (`EXECUTE_TESTS_ON=master pytest`)

Also the following steps were realized (if applies):
- [ ] Use in-line comments to explain your code
- [ ] Write docstrings to your code
- [ ] For new functionalities: Explain in readthedocs
- [ ] Write test(s) for your new patch of code

*Please mark above checkboxes as following:*
- [ ] Open
- [x] Done

In case of an error due to linting, run `black . --exclude docs/` and push your changes.
Note that in case you do not fix a whole issue you should start your PR with `Address #xyz`.

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl-institut/mvs_eland/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

File: pvcompare/README.txt [hunk 1]
```diff
@@ -21,9 +21,9 @@ thermal components on corresponding KPIs.
 	b. ~/[...]/pvcompare/pvcompare/data/user_inputs/mvs_inputs_sector_coupling/ in case of simulating sector coupling scenarios A, B, C and D
```
Comment (SabineHaas) [inline pvcompare/README.txt]: In our paper there's no sector-couploed scenario D. Is there one in your thesis?

Comment (MaGering) [reply pvcompare/README.txt]: No there isn't either. But there is one on `run_sector_coupling_scenarios`. It is for simulating warm water.

**Comment by MaGering** [comment]
I had to add the README as a markdown file to the `with_battery` directory. RST did not work somehow.

**Comment by SabineHaas** [comment]
Do I have to do anything here now? There is a file named old_readme and I am not sure of the purpose.

**Comment by SabineHaas** [comment]
The build already fails for your branch @MaGering by the way, I have triggered the build manually: https://github.com/greco-project/pvcompare/actions
As we are not going to merge it into `dev` I dont see a problem here.

**Comment by MaGering** [comment]
> Do I have to do anything here now? There is a file named old_readme and I am not sure of the purpose.

@SabineHaas, right thanks! :) I corrected that. 


> The build already fails for your branch @MaGering by the way, I have triggered the build manually: https://github.com/greco-project/pvcompare/actions
> As we are not going to merge it into `dev` I dont see a problem here.


I unfortunately couldn't fix this error on the branch without investing much time to adapt the test. But since we're not going to merge this branch I agree leaving it like this. 

So I guess this branch is ready for merging, if you don't have further adjustments for the README.md or the README.rst in pvcompare directory.

**Pull request merged**